core/generated/main.go

```diff
@@ -79,7 +104,8 @@ func main() {
 	g.GenerateModel("contents")
 	g.GenerateModel("content_deals")
 	g.GenerateModel("auth_tokens")
-	g.GenerateModel("autoretrieves")
+	// ? Need to manually specify Autoretrieves otherwise it will use "autoretriefes" model name
```

Comment:
@alvin-reyes this is super weird - I have no idea why the model gets named "Autoretriefs" if this is not specified

Reply:
@elijaharita let me know it's GORM trying to convert the table to be singular from plural so this override is necessary 🤦‍♂️